CMakeLists.txt

```diff
@@ -153,6 +153,9 @@ elseif(CMAKE_C_COMPILER_ID MATCHES "MSVC")
         /wd4820 # Disable: bytes padding added after data member
         /wd4996 # Disable: potentially unsafe stdlib methods
         /wd5045 # Disable: Compiler will insert Spectre mitigation for memory load if /Qspectre switch specified
+        # See https://github.com/AOMediaCodec/libavif/pull/438
+        # Error when attempting to print non-ASCII character to console
+        /source-charset:.65001 /execution-charset:.20127
```

Comment:
1. Could you change this to
         /source-charset:utf-8 /execution-charset:us-ascii
?

If this form works, it is easier to understand than the code page identifiers as decimal numbers.

2. Could you also add a comment to explain why /source-charset:.65001 /execution-charset:.20127 is better than /utf-8?

Details:
I found this Chromium issue on using the /utf-8 compiler flag on Windows:
https://bugs.chromium.org/p/chromium/issues/detail?id=637203

and the changelist:
https://codereview.chromium.org/2543743002

I verified that the /utf-8 flag is still in the current Chromium source tree:
https://source.chromium.org/chromium/chromium/src/+/master:build/config/win/BUILD.gn;l=64?q=%22%2Futf-8%22&ss=chromium&start=51

I also found the /utf-8 flag used by two third-party libraries used by Chromium: re2 and harfbuzz-ng. So the /utf-8 flag seems like a well-known solution to this problem. It would be good to explain why /source-charset:.65001 /execution-charset:.20127 is a superior solution.

Reply:
If those strings that contains unicode is not going to be printed to console using `printf` then use /utf-8 flag is fine. Both re2 and harfbuzz are text-processing related libraries, so /utf-8 precisely fit their need (note that /execution-charset also specify the encoding used to store strings in executable). But for command-line tools' messages, it's better to use ascii only - even if you use `SetConsoleOutputCP` WinAPI to change current code page to UTF-8, there's still a chance that the user's font can't display it.